flink-python/pyflink/fn_execution/stream_fast.pyx

```diff
@@ -94,6 +100,21 @@ cdef class OutputStream:
         self.buffer_size = 1024
         self.pos = 0
 
+    cpdef void write(self, bytes v):
```

Comment:
Why it's cpdef and the other methods are declared cdef?

Reply:
This method could be called in pure python function e.g. `ResettableIO` which is used in `ArrowCoderImpl`